eventhubs/client/azure-eventhubs/src/main/java/com/azure/eventhubs/PartitionProperties.java

```diff
@@ -83,12 +89,12 @@ public String lastEnqueuedOffset() {
     }
 
     /**
-     * Gets the time of the last enqueued message in the partition's stream.
+     * Gets the date time, in UTC, of the last enqueued message in the partition's stream.
```

Comment:
Gets the *instant*. This would be consistent with `propertyRetrievalTime` below.